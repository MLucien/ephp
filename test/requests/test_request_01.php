<?php

$_REQUEST["a"] = 12;
$_REQUEST["b"] = 24;

var_dump($_REQUEST);
