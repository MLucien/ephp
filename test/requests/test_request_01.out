array(2) {
  ["a"]=>
  int(12)
  ["b"]=>
  int(24)
}
